### PR TITLE
Ensure dirs exist before using them

### PR DESCRIPTION
**Pull request by tylerwilliams**

### CODE REVIEW

File: enterprise/server/backends/pebble_cache/pebble_cache.go [hunk 1]
```diff
@@ -103,9 +103,8 @@ func Register(env environment.Env) error {
 	if *rootDirectory == "" {
 		return nil
 	}
-	if env.GetCache() != nil {
-		log.Warning("A cache has already been registered, skipping registering pebble_cache.")
```
Comment (vadimberezniker) [inline enterprise/server/backends/pebble_cache/pebble_cache.go:107]: Was this removal intentional?

**Review by vadimberezniker — APPROVED** [review]

File: enterprise/server/backends/pebble_cache/pebble_cache.go (hunk 1 above)
Comment (tylerwilliams) [reply enterprise/server/backends/pebble_cache/pebble_cache.go:107]: yup!
